site.local.yml

```diff
@@ -0,0 +1,70 @@
+#
+# This is an example Antora playbook configuration file. By default it's not
+# the one that's recommended in the documentation. It's included as a sample or
+# demonstration of what is possible, along with links to the relevant sections
+# of the documentation.
+#
+
+#
+# In this section, you can configure the core playbook configuration settings.
+# This section covers the site's title, base URL, and site start page. For more
+# information, see https://docs.antora.org/antora/1.0/playbook/configure-site/
+#
+site:
+  title: ownCloud Documentation
+
+content:
+  sources:
+  # This configuration enables what's referred to as "Author Mode". In this
+  # "mode", the local working copy is used to build the documentation. It
+  # doesn't refer to a specific branch, whether local or remote. This avoids
+  # generating only static assets, but no documentation. 
+  # For more information, see https://docs.antora.org/antora/1.0/playbook/author-mode/. 
+  # Normally, you would add standard content sources here. For more information
+  # about them, see https://docs.antora.org/antora/1.0/playbook/configure-content-sources/.
+  - url: ./
+    branches: HEAD
+  # Other components (modules) that you wish to include as part of this playbook.
+  - url: https://github.com/owncloud/android.git
+    branches:
+    - master-antora
+    start_path: docs/
+  - url: https://github.com/owncloud/ios.git
+    branches:
+    - master-antora
+    start_path: docs/
+  - url: https://github.com/owncloud/client.git
+    branches:
+    - master-antora
+    start_path: docs/
+
+#
+# In this section, you can configure the UI/theme for the documentation. This
+# covers the UI theme bundle to use, and whether to use a default page layout.
+# For more information, see
+# https://docs.antora.org/antora/1.0/playbook/configure-ui/
+#
+ui:
+  bundle:
+    url: https://github.com/owncloud/docs-ui/releases/download/1.0.0/ui-bundle.zip
```

Comment:
Why do we need ``releases`` instead using the directory ``bundle`` which exists and would be easy to un-gitignore (...gulp pack)? I may be wrong, but this approach implies a manual steep creating a release by someone after we have made a PR, commit, approval, merge process. Why a second process ?

Reply:
It was where I was at at the time that I originally created the file. @tboerger's implementing a build pipeline for the docs. I'll change this after he's finished.

Reply:
Let us not forget to update this accordingly :smile: